src/NuGet.Services.ServiceBus/NuGet.Services.ServiceBus.csproj

```
@@ -12,11 +12,12 @@
     <PackageReference Include="Microsoft.Extensions.Logging">
       <Version>2.2.0</Version>
     </PackageReference>
+    <PackageReference Include="Microsoft.Identity.Client" Version="4.25.0" />
```

Could we follow the style of the other `PackageReference`s in this file? To be honest, I personally prefer your style but we should stay consistent with the project's style.

```suggestion
    <PackageReference Include="Microsoft.Identity.Client">
        <Version>4.25.0</Version>
    </PackageReference>
```